duckduckgo/iOS
# lock screen widgets

Task/Issue URL: https://app.asana.com/0/414709148257752/1204600823062806/f
Tech Design URL:
CC:

**Description**:

Adds some initial Lock Screen widgets to the app.

* Favorites - launches a new tab showing the home page
* Fire Button - launches the app and runs the fire button
* Email Protection - if signed in creates an email address and puts it in on the pasteboard, otherwise launches the email protection web page
* Search - launches a new tab and focuses for a search
* Voice - launches a new tab using voice search

Refactored app deep links into an enum.

Removed default browser pixel as not used at this point.

Refactored email manager to be in the main view controller.

**Steps to test this PR**:
1. Lock the device and customise the screen.
2. Add all the widgets and test them.
3. Check the copy matches.
4. Check about screen link still works.
5. Check email protection (sign in, sign out, autofill, etc)



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
